lua/lv-which-key/init.lua

```diff
@@ -61,54 +61,15 @@ local opts = {
   nowait = false, -- use `nowait` when creating keymaps
 }
 
--- no hl
-vim.api.nvim_set_keymap("n", "<Leader>h", ':let @/=""<CR>', { noremap = true, silent = true })
-
--- explorer
-
-vim.api.nvim_set_keymap(
-  "n",
-  "<Leader>e",
-  ":lua require'lv-nvimtree'.toggle_tree()<CR>",
-  { noremap = true, silent = true }
-)
-
-vim.api.nvim_set_keymap("n", "<Leader>f", ":Telescope find_files<CR>", { noremap = true, silent = true })
-
--- dashboard
-vim.api.nvim_set_keymap("n", "<Leader>;", ":Dashboard<CR>", { noremap = true, silent = true })
-
--- Comments
-vim.api.nvim_set_keymap("n", "<leader>/", ":CommentToggle<CR>", { noremap = true, silent = true })
-vim.api.nvim_set_keymap("v", "<leader>/", ":CommentToggle<CR>", { noremap = true, silent = true })
-
--- close buffer
-vim.api.nvim_set_keymap("n", "<leader>c", ":BufferClose<CR>", { noremap = true, silent = true })
-
--- Save
-vim.api.nvim_set_keymap("n", "<leader>w", ":w!<CR>", { noremap = true, silent = true })
-
--- Quit
-vim.api.nvim_set_keymap("n", "<leader>q", ":q!<CR>", { noremap = true, silent = true })
-
--- open lv-config
-vim.api.nvim_set_keymap(
-  "n",
-  "<leader>.",
-  ":e " .. CONFIG_PATH .. "/lv-config.lua<CR>",
-  { noremap = true, silent = true }
-)
-
 local mappings = {
-  ["w"] = "Save",
-  ["q"] = "Quit",
-  ["."] = "LunarConfig",
-  ["/"] = "Comment",
-  ["c"] = "Close Buffer",
-  ["e"] = "Explorer",
-  ["f"] = "Find File",
-  ["h"] = "No Highlight",
-  [";"] = "Dashboard",
+  ["w"] = { ":w!<CR>", "Save" },
```

Comment:
Use \<cmd> here instead of : (consistency)

Reply:
Okay. Changed to using `<cmd>`!